attestation-evidence-broker/docs/design/design.md

```diff
@@ -0,0 +1,121 @@
+This document provides that how to use Attestation Evidence Broker (AEB) which is a host service to satisfy the requests from guest for getting **SEV(-ES) runtime remote attestation report**.
+
+# Background of SEV(-ES) runtime remote attestation
+
+Compared with SEV SNP and Intel TDX, the runtime remote attestation of SEV(-ES) does NOT support guest-provided data to be included into the attestation report [1,2]. In addition, SEV(-ES) does NOT support dynamic measurement and can only verify static measurement values, which are always constant values. Although SEV(-ES) has this limitation, runtime remote attestation is still useful. When the SEV(-ES) guest is running, it can report the attestation evidence with fixed launch digest as a heartbeat for trusted healthy check.
```

Comment:
s/SEV SNP/SEV-SNP/

Reply:
I remember we talked about what the differences are between attestation evidence and attestation report. Here `attestation report` should be used.